protocols/relay/src/behaviour/handler.rs

```diff
@@ -380,6 +387,11 @@ pub struct Handler {
     alive_lend_out_substreams: FuturesUnordered<oneshot::Receiver<()>>,
     /// Futures relaying data for circuit between two peers.
     circuits: Futures<(CircuitId, PeerId, Result<(), std::io::Error>)>,
+
+    inbound_requests_futs:
+        FuturesUnordered<BoxFuture<'static, Result<Event, inbound_hop::FatalUpgradeError>>>,
+    stop_requested_streams: VecDeque<StopCommand>,
+    outbound_stop_futs: FuturesUnordered<BoxFuture<'static, Result<Event, FatalUpgradeError>>>,
```

Comment:
Same here, I'd prefer us to be consistent in prefixing the module for types like `FataUpgradeError`.